### PR TITLE
Improved print_signatories_by_position to add more use cases

### PR DESCRIPTION
**Pull request by duchenean**
Hello, 
I've added some tests to print_signatories_by_position and I've improved it to handle more use cases.

### CODE REVIEW

**Comment by gbastien** [comment]
@duchenean 

LGTM mais regardé vite fais, second avis de @odelaere serait bienvenu ;-)

Gauthier

File: src/Products/PloneMeeting/browser/views.py [hunk 1]
```diff
@@ -1316,40 +1316,56 @@ def print_signatures_by_position(self, **kwargs):
...
     def print_signatories_by_position(self,
-                                      signature_format=(u'position_type', u'person'),
-                                      separator=u', ',
-                                      ender=''):
+                                      signature_format=(u'prefixed_position_type', u'person'),
+                                      separator=u',',
+                                      ender=u''):
         """
         Print signatories by position
+        :param signature_format: can be 'position_type', 'prefixed_position_type', 'person',
```
Comment (odelaere) [inline src/Products/PloneMeeting/browser/views.py:1324]: Not clear what's the difference between prefixed_position_type and position_type.
Please add a bit more explanation for every possible values of signature_format parameter and make it so the differences between them are obvious

File: src/Products/PloneMeeting/browser/views.py [hunk 2]
```diff
@@ -1316,40 +1316,56 @@ def print_signatures_by_position(self, **kwargs):
...
+        'person_title', 'secondary_position_type', 'prefixed_secondary_position_type',
+        an attribute of PMHeldPosition or a str (in this case it just print the str)
+        :param separator: str that will be appended at the end of each line (except the last one)
```
Comment (odelaere) [inline src/Products/PloneMeeting/browser/views.py:1327]: Not clear what's the difference between prefixed_position_type and position_type.
Please add a bit more explanation for every possible values of signature_format parameter and make it so the differences between them are obvious

File: src/Products/PloneMeeting/tests/testContacts.py [hunk 3]
```diff
@@ -475,6 +475,77 @@ def test_pm_MeetingGetItemAbsents(self):
         self.assertEqual(item1.getItemAbsents(), (hp_uid, ))
         self.assertFalse(item2.getItemAbsents())
 
+    def test_pm_print_signatories_by_position(self):
```
Comment (odelaere) [inline src/Products/PloneMeeting/tests/testContacts.py:478]: Please hardcode the expected value in asserts. So we know if somthing breaks the formatting

Comment (duchenean) [reply src/Products/PloneMeeting/tests/testContacts.py:478]: Will do, thanks :+1:

File: src/Products/PloneMeeting/browser/views.py (hunk 2 above)
Comment (duchenean) [reply src/Products/PloneMeeting/browser/views.py:1334]: Done :ok_hand:

File: src/Products/PloneMeeting/tests/testContacts.py (hunk 3 above)
Comment (duchenean) [reply src/Products/PloneMeeting/tests/testContacts.py:478]: Done :smiley:

**Pull request merged**